web/crux/templates/vaultwarden.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Vaultwarden",
+  "description": "Alternative implementation of the Bitwarden server API written in Rust and compatible with upstream Bitwarden clients, perfect for self-hosted deployment where running the official resource-heavy service might not be ideal.",
+  "registries": [
+    {
+      "name": "Vaultwarden / Docker HUB",
+      "type": "hub",
+      "hub": {
+        "imageNamePrefix": "vaultwarden"
+      }
+    }
+  ],
+  "images": [
+    {
+      "name": "vaultwarden-server",
+      "registryName": "Vaultwarden / Docker HUB",
+      "image": "server",
+      "tag": "latest",
+      "config": {
+        "name": "vaultwarden",
+        "ports": [
```

Comment:
Websocket is also an option for that port 3012 is used, we might not be able to deploy that correctly with crane's current ingress controller features, without modification,
It is possible with dagent though, using custom labels. 
We need a rule like:
```
Host(`vault.dyrectorio.org`) && PathPrefix(`/notifications/hub/negotiate`)
```
That routes to port 3012, all else can go to 80.

Reply:
Custom labels for Traefik to do the custom routing are missing.

Reply:
I don't know Traefik labels.